include/torch-mlir/Dialect/Torch/Transforms/Passes.h

```diff
@@ -32,11 +32,20 @@ struct TorchLoweringPipelineOptions
   // If this option is false, only do the bare minimum for correctness.
   Option<bool> optimize{*this, "optimize", llvm::cl::desc("Do optimizations."),
                         llvm::cl::init(true)};
-  
+
   // If this option is false, decompose complex operations.
   // If this option is true, skip decomposition of complex operations.
-  Option<bool> decompose{*this, "decompose-complex-ops", llvm::cl::desc("Decompose complex operations."),
-                        llvm::cl::init(true)};                      
+  Option<bool> decompose{*this, "decompose-complex-ops",
```

Comment:
I would prefer that we have something a bit more principled here. Like one option which is an allowlist and another which is a denylist so users can control it more precisely.